recipes/neo4j-connector/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "neo4j-connector" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/textkernel/neo4j-connector/archive/v1.0.1.tar.gz
```

Could you please change it to?
```suggestion
  url: https://github.com/textkernel/neo4j-connector/archive/v{{ version }}.tar.gz
```
Because in that way when we have new version os this package, our bot will be able to update it easily